Server/ChatService/routes/calls.py

```
@@ -0,0 +1,360 @@
+                    "recipient_user_id": int(request_body.callee_id),
+                    "call_id": call_data["call_id"],
+                    "caller_id": user_id,
+                    "call_type": request_body.call_type,
+                }
+                response = await client.post(
+                    f"{user_service_url.rstrip('/')}/api/v1/users/notify/call-incoming",
+                    headers=auth_header,
+                    json=fcm_payload,
+                )
+                print(f"[calls.create] FCM notification response: {response.status_code}")
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)